hathor/crypto/util.py

```diff
@@ -227,6 +226,8 @@ def get_address_from_redeem_script_hash(redeem_script_hash: bytes,
         :return: address in bytes
         :rtype: bytes
     """
+    settings = get_settings()
+    version_byte = version_byte if version_byte is not None else settings.MULTISIG_VERSION_BYTE
```

Comment:
Same here.

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/769#discussion_r1320896387

Reply:
Thanks!